types/core/speedy-media-source.d.ts

```diff
@@ -30,7 +30,7 @@ export class SpeedyMediaSource {
      * The underlying wrapped object
      * @returns {SpeedyMediaSourceNativeElement}
      */
-    get data(): SpeedyMediaSourceNativeElement;
```

Comment:
I don't know why the typescript compiler convert to any, instead it should be SpeedyMediaSourceNativeElement